backend/reservations/reporting.py

```diff
@@ -139,3 +139,69 @@ def hotel_export(hotel):
 
     egest_csv(rows, fields, hotel_export_file)
     return hotel_export_file
+
+
+def rooming_list_export(hotel):
+    rooms = Room.objects.filter(name_hotel = hotel.title())
+    guests = Guest.objects.filter(name_hotel = hotel.title())
+    if rooms.count() == 0:
+        raise Exception("No rooms found for hotel %s" % hotel)
+    
+    cols = [
```

Comment:
Given the nature of manual swaps we are looking at, we should also include

* `Room.guest_notes` (I'm assuming that `Room.notes` stays with the room but maybe that is a false assumption?)
* `Room.is_art` (we are allowing the movement of art rooms, can we safely assume it will always be between art floors? asking this in slack....)
* `is_placed` seems like it would be helpful to verify that rooms are in sync? maybe superfluous tho...

Reply:
is_placed seems superfluous. added the others